lib/extensions/polyglot_with_database/coursemology/polyglot/language/python/python2_point7.rb

```diff
@@ -0,0 +1,7 @@
+module Extensions::PolyglotWithDatabase::Coursemology::Polyglot::Language::Python::Python2Point7
```

Comment:
@lowjoel Actually how about do this in a configuration file(instead of extensions) ?  So that we have a configuration file for polyglot in our main app, and it tells polyglot how to interact with AR.